src/test_common.rs

```diff
@@ -53,22 +59,6 @@ impl Algorithm<Item> for XOR128 {
     }
 }
 
-impl fmt::UpperHex for XOR128 {
```

Comment:
Why did you remove that formatting implementation?

Reply:
We used it only for `test_hasher_light` where we hashed strings. Now this test is rewritten and it uses byte vectors which seems to be standard practice in crypto (passing strings to cryptographic algorithms always requires dealing with encoding which is actually additional headache). Apart from that I didn't actually see where this logic is needed

Reply:
As `XOR128` is only used for testing and not part of the public interface it's OK to remove that debug formatting.